src/caustics/cosmology.py

```diff
@@ -43,12 +43,10 @@ class Cosmology(Parametrized):
     This class provides an interface for cosmological computations used in lensing
     such as comoving distance and critical surface density.
 
-    Units
-    -----
     Distance
-        Mpc
+        *Unit: Mpc*
     Mass
-        solar mass
+        *Unit: SolMass*
```

Comment:
hould this be `solMass` or `SolMass` for consistency?

Reply:
solMass is maintained throughout